.pre-commit-config.yaml

```
@@ -28,10 +28,20 @@ repos:
       - id: requirements-txt-fixer
       - id: mixed-line-ending
         args: ["--fix=auto"]
-  - repo: https://github.com/google/yapf
-    rev: v0.43.0
+  - repo: https://github.com/psf/black
+    rev: 23.11.0
     hooks:
-      - id: yapf
+    - id: black
+      language_version: python3
+  - repo: https://github.com/pycqa/isort
+    rev: 5.12.0
+    hooks:
+    - id: isort
+      args: ["--profile", "black", "--filter-files"]
```

Personally, I like the "Vertical Hanging Indent" (3) mode for the "Multi-Line Output Modes" option of isort over the default "Grid" (0) mode. Here is a list of the modes available: https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html. (Again, not sure if the syntax is 100% correct, so please be sure to test it)

```suggestion
    - id: isort
      args: ["--profile", "black", "--filter-files", "--multi-line", "3"]
```